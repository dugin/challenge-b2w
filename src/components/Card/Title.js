import React from 'react';
import styled from 'styled-components';
import colors from "../../styles/constants/colors";
import PropTypes from "prop-types";


const Wrapper = styled.h1`  
      color: ${colors.brightTurquoise};
      font-weight: 300;
      padding-bottom: 10px;
      border-bottom: 1px solid ${colors.pattensBlue};
`;

const Title = (props) => {

    return (
        <Wrapper>
            {props.text}
        </Wrapper>
    )
};

Title.propTypes = {
    text: PropTypes.string.isRequired,
};
export default Title;