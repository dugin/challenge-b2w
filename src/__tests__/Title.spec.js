import React from 'react';
import toJson from 'enzyme-to-json';
import {shallow} from 'enzyme';
import Title from "../components/Card/Title";

describe('Card Component', () => {

    const title = shallow(<Title/>);

    it('renders properly', () => {
        expect(toJson(title)).toMatchSnapshot();
    });
});