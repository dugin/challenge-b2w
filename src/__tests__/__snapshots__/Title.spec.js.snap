// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component renders properly 1`] = `
<styled.h1>
  Coruscant
</styled.h1>
`;
